packages/template-retail-react-app/app/ssr.js

```diff
@@ -329,7 +329,7 @@ const {handler} = runtime.createHandler(options, (app) => {
     )
 
     // Handle the redirect from SLAS as to avoid error
-    app.get('/callback?*', (req, res) => {
+    app.get('/callback', (req, res) => {
```

Comment:
This change isn't quite related to the base path work. What the `?` here is doing is making the `k` optional so `/callbac` (without the k) is a valid route which I don't think we want.